src/components/controls/TransitToggleButtonBar.tsx

```diff
@@ -8,6 +8,7 @@ interface ITtransitToggleButtonBarProps {
     selectedTransitTypes: TransitType[];
     toggleSelectedTransitType?: (type: TransitType) => void;
     disabled?: boolean;
+    blurred?: boolean;
```

Comment:
I'm not convinced you need both disabled & blurred

If you use pointer-events: none; (at TransitToggleButton.tsx), these changes can be reverted I think

Reply:
They are different things. In the start view we want the transittoggles to be both disabled, but also blurred so people understand that it is not active. This is different from what we want in link view, in link view we want to show the transit type for the user, so we want disabled toggles, but not blurred.